src/main/java/hanieum/conik/adapter/user/persistence/UserEntity.java

```diff
@@ -0,0 +1,40 @@
+package hanieum.conik.adapter.user.persistence;
+
+import hanieum.conik.domain.user.UserRole;
+import hanieum.conik.global.domain.BaseEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.NaturalId;
+
+@Entity
+@Data
+@Table(name = "User")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class UserEntity extends BaseEntity {
```

Comment:
위에 의견과 동일해요
아마 근데 User로 만들면 JWT에서 사용하는 내장 User와 겹쳐서 문제가 생길거라 member나 다른 걸로 변경해야겠군요
```suggestion
public class User extends BaseEntity
```

Reply:
넵 일단 초기에 User로 되어있어서 그대로 가져가기는 했는데 member로 수정하는게 좋을 것 같아요